lib/fluent/plugin/out_lm.rb

```diff
@@ -31,7 +31,10 @@ class LmOutput < BufferedOutput
 
     config_param :compression,  :string, :default => ""
 
-    $versionId = "0.0.1"
+    config_param :log_source,  :string, :default => "fluentd"
```

Comment:
It should default to `lm-logs-fluentd` perhaps?

Reply:
Done